core-war/src/main/java/org/silverpeas/web/pdc/control/PdcSearchSessionController.java

```diff
@@ -118,7 +118,7 @@
   // pagination de la liste des resultats (search Engine)
   private int indexOfFirstResultToDisplay = 0;
   private String displayParamChoices = null; // All || Res || Req
-  private int nbResToDisplay = -1;
+  private int nbResToDisplay = 10;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Assign this magic number 10 to a well-named constant, and use the constant instead. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS109)